### DISCUSSION TITLE
Malformed URI "http://". ERROR In Request.php line 303

### DISCUSSION THREAD

**Discussion by jigneshflair** [laravel/framework#52142] [Q&A] [answered]
### Laravel Version

11.0

### PHP Version

8.2

### Database Driver & Version

MySql

### Description

I have updated the existing laravel 10 project to laravel 11.
and update all the dependency ( as per the document guide suggest here https://laravel.com/docs/11.x/upgrade ).
and i have run  *composer update*  command after removing the existing vendor folder and composer.lock file.
then this below issue arises.

Malformed URI "http://". ERROR In Request.php line 303

.env and config file also have the full url but i don't know why this error brake the project.

![image](https://github.com/user-attachments/assets/9c1797fb-bb58-4fde-9f94-1a7f0cc12606)

please let me know if any one know about this error or how to solve this.

Thanks in advance.

### Steps To Reproduce

- Change the existing dependency for laravel 10  to  laravel 11 as per the laravel 11 update guide  document suggest.
- Run *Composer update*.

**Comment by noefleury** [comment, marked as answer]
Hello, I doubt that it's a Laravel 11 issue.

This is what's happening during the [Laravel bootstrapping](https://github.com/laravel/framework/blob/11.x/src/Illuminate/Foundation/Bootstrap/SetRequestForConsole.php#L18), and more precisely during your `composer update` :

```php
$uri = $app->make('config')->get('app.url', 'http://localhost');
```

And then your error happen later during the [Request creation](https://github.com/symfony/http-foundation/blob/7.1/Request.php#L303) in `symfony/http-foundation`.

I suggest you to double-check the `url` property in `config/app` file.
